jdi-light-material-ui-tests/src/main/java/io/github/com/pages/navigation/StepperFrame.java

```diff
@@ -8,4 +8,25 @@
 
     @JDIStepper(root = "#root")
     public Stepper stepper;
+
+    @JDIStepper(stepperTitle = ".MuiTypography-root.MuiTypography-body1")
+    public Stepper stepperTitle;
+
+    @JDIStepper(stepperNextButton = "//*[text()='Next']/..")
+    public Stepper stepperNextButton;
+
+    @JDIStepper(stepperBackButton = "//*[text()='Back']/..")
+    public Stepper stepperBackButton;
+
+    @JDIStepper(stepperFinishButton = "//*[text()='Finish']/..")
+    public Stepper stepperFinishButton;
+
+    @JDIStepper(stepperResetButton = "//*[text()='Reset']/..")
+    public Stepper stepperResetButton;
+
+    @JDIStepper(stepperSkipButton = "//*[text()='Skip']/..")
+    public Stepper stepperSkipButton;
+
+    @JDIStepper(stepperCompleteStepButton = "//*[text()='Complete Step']/..")
+    public Stepper stepperCompleteButton;
```

Comment:
Are really annotations suitable for stepper? Isn't enough simple locators?

Reply:
In future it's planning to have if it's possible all controls on one page for each element, so annotations could be suitable, no sense to update now then